dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/HostAPITest.java

```diff
@@ -842,5 +847,359 @@ public void Test_SiteKey_Valid_DNS_Format_No_Failure_Expected()  {
         }
     }
 
+    @Test
+    public void getSitebyItsIdFromContentlet() throws Exception {
```

Comment:
Add doc to all of methods
/**
* Method to test: include this in case the method to test does not belong to the class this test-class is testing 
* Given Scenario: Scenario under test
* ExpectedResult: Expected result that the assertion is checking against
*
*/